.pre-commit-config.yaml

```diff
@@ -44,11 +42,17 @@ repos:
   - repo: https://github.com/PyCQA/bandit
     rev: 1.7.4
     hooks:
-    -   id: bandit
+      - id: bandit
         args: ["--ini", ".bandit", "--recursive"]
 
-  - repo: https://github.com/terrencepreilly/darglint
-    rev: v1.8.1
-    hooks:
-    - id: darglint
-      exclude: ^(tests|scripts)/
+
+  # - repo: https://github.com/codespell-project/codespell
+  #   rev: "v2.2.2"
+  #   hooks:
+  #     - id: codespell
+
+  # - repo: https://github.com/terrencepreilly/darglint
```

Comment:
should we just remove that instead of keeping it commented ?

Reply:
let's remove them you are right